### PR TITLE
feat: expose entire affinity field for native nats eventbus

### PR DESCRIPTION
**Pull request by whynowy**
Signed-off-by: Derek Wang <whynowy@gmail.com>

Checklist:

* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo-events/blob/master/USERS.md).

### CODE REVIEW

**Review by alexec — COMMENTED** [review]
please set-up 30m to review

File: pkg/apis/eventbus/v1alpha1/types.go [hunk 1]
```diff
@@ -63,9 +63,10 @@ var (
 // NativeStrategy indicates to install a native NATS service
 type NativeStrategy struct {
 	// Size is the NATS StatefulSet size
-	Replicas     int32         `json:"replicas,omitempty" protobuf:"varint,1,opt,name=replicas"`
-	Auth         *AuthStrategy `json:"auth,omitempty" protobuf:"bytes,2,opt,name=auth,casttype=AuthStrategy"`
-	AntiAffinity bool          `json:"antiAffinity,omitempty" protobuf:"varint,3,opt,name=antiAffinity"`
+	Replicas int32         `json:"replicas,omitempty" protobuf:"varint,1,opt,name=replicas"`
+	Auth     *AuthStrategy `json:"auth,omitempty" protobuf:"bytes,2,opt,name=auth,casttype=AuthStrategy"`
+	// Deprecated, use Affinity instead, will be removed in v1.5
```
Comment (whynowy) [inline pkg/apis/eventbus/v1alpha1/types.go:68]: Deprecated: xxx

**Review by alexec — DISMISSED** [review]

File: pkg/apis/eventbus/v1alpha1/types.go (hunk 1 above)
Comment (whynowy) [reply pkg/apis/eventbus/v1alpha1/types.go:69]: Changing to `Deprecated:` will cause lint failure at the place this field being referenced - it does not allow anybody to reference a deprecated property.

**Comment by whynowy** [comment]
@alexec - Could you approve it again?

**Review by alexec — APPROVED** [review]

**Pull request merged**